### PR TITLE
{math}[GCCcore/11.3.0] PARI-GP v2.15.4

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/PARI-GP/PARI-GP-2.15.4-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+skipsteps = ['configure']
+
+prebuildopts = './Configure --prefix=%(installdir)s &&'
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['gp', 'gp-2.15', 'gphelp', 'tex2mail']] +
+             ['include/pari/%s' % x for x in ['pari.h', 'genpari.h']],
+    'dirs': ['bin', 'include']
+}
```
Comment (boegel) [inline easybuild/easyconfigs/p/PARI-GP/PARI-GP-2.15.4-GCCcore-11.3.0.eb:34]:
```suggestion
}

sanity_check_commands = ["gp --help"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/8004b024ad1d4085794b9393b4b3ce8a for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**